pytorch_lightning/loggers/base.py

```diff
@@ -5,6 +5,10 @@
 from argparse import Namespace
 from functools import wraps
 from typing import Union, Optional, Dict, Iterable, Any, Callable, List, Sequence, Mapping, Tuple
+try:
+    from omegaconf import OmegaConf, DictConfig
+except ImportError:
+    pass
```

Comment:
you should add note about installing missing as other loggers..

Reply:
As using hydra is optional, I don't want to raise an exception `ImportError` here. How do you think about:
```suggestion
    import logging
    logging.debug('Hydra is not installed. If you want to use Hydra for managing configuration, please install it with `pip install hydra_core`')
```

Reply:
true but then you shall add
```
try:
    import ...
except ...:
    ...
    USING_HYDRA_CONF = True
else:
    USING_HYDRA_CONF = True
```